open-api-spec.yaml

```diff
@@ -451,6 +467,177 @@ paths:
           in: query
           name: cursor
           description: The cursor key to list from
+  /quotes:
+    post:
+      summary: Create a Quote
+      tags: []
+      operationId: create-quote
+      responses:
+        '201':
+          description: Quote Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/quote'
+              examples:
+                New Fixed Send Quote for $25:
+                  value:
+                    id: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+                    accountId: 'https://openpayments.guide/alice/'
+                    receivingAccount: 'https://openpayments.guide/bob/'
+                    sendAmount:
+                      value: '2500'
+                      assetCode: USD
+                      assetScale: 2
+                    receiveAmount:
+                      value: '2198'
+                      assetCode: EUR
+                      assetScale: 2
+                    createdAt: '2022-03-12T23:20:50.52Z'
+                    expiresAt: '2022-04-12T23:20:50.52Z'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
```

Comment:
What should the response be if Rafiki's request to wallet doesn't succeed (such as if the sender's account doesn't have sufficient funds for the quoted amount)? 403 or whatever code the wallet returns? (What should the wallet return if the user doesn't have funds?)

Reply:
We discussed returning `5XX`